### PR TITLE
Account update

### LINKED ISSUE #2286: Add a note on session revocation
The sign-in page offers the possibility to the user to select whether he wants to stay signed-in or not. 

Screenshot 2020-03-31 at 13 25 05

**To do** - Provide in the Learning Center quick note on what the behavior is around session revocation.
**Where** - Postman > Launching Postman > Postman account > [Signing up for a Postman account](https://learning.postman.com/docs/postman/launching-postman/postman-account/#signing-up-for-a-postman-account)

**Content** - 
- If the checkbox is checked, the session expires after 30 days - you will need to re-authenticate every 30 days to continue working.
- If not, it expires after 30 minutes - you will need to re-authenticate every 30 minutes to continue working.

### PR DESCRIPTION
**Pull request by SueSmith**
Fixes #2286 

cc @cyrielle-hada @claireriley this page was woefully out of date so just gave it a general update

### CODE REVIEW

File: src/pages/docs/postman/launching-postman/postman-account.md [hunk 1]
```diff
@@ -25,119 +25,111 @@ warning: false
...
 
-When you sign up for a free Postman account, you can:
+Signing up for an account is optional—you can continue to use Postman without an account if you prefer. Check out the [EULA](https://www.postman.com/licenses/postman_base_app), [security overview](https://www.postman.com/security), and [privacy policy](https://www.postman.com/licenses/privacy).
 
-* [Sync](/docs/postman/launching-postman/syncing/) and back up your history, collections, workspaces, environments, and header presets.
-* Easily work on multiple Postman instances from different machines.
-* Create [collection links](/docs/postman/collections/sharing-collections/) to send to other developers.
-
-Signing up is completely optional. Read the [EULA](https://www.postman.com/licenses/postman_base_app), [security page](https://www.postman.com/security), and [privacy page](https://www.postman.com/licenses/privacy) to learn more.
+* [Signing up for a Postman account](#signing-up-for-a-postman-account)
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/launching-postman/postman-account.md:32]: d'oh 🤦‍♀

File: src/pages/docs/postman/launching-postman/postman-account.md [hunk 2]
```diff
@@ -25,119 +25,111 @@ warning: false
...
+![App Sign In](https://assets.postman.com/postman-docs/pm-app-sign-in.jpg)
 
-Click the Pencil icon.
+Enter your account email, check the __I'm not a robot__ box, and click __Submit__.
 
-Upload or drag and drop the image you want for your profile from your computer and click the **Upload** button.
+![Trouble Signing In](https://assets.postman.com/postman-docs/trouble-signing-in.jpg)
 
-[![upload](https://assets.postman.com/postman-docs/WS-upload-image1.png)](https://assets.postman.com/postman-docs/WS-upload-image1.png)
+Postman will send you an email with instructions for recovering access to your account.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/launching-postman/postman-account.md:133]: cool added a generic note re if you still have problems logging in contact support - posted in product support slack channel to see if anyone has a suggested workaround so will update if anyone comes back with anything.. 👍

[the remark this answers is not in the document]
Comment (cyrielle-hada) [reply src/pages/docs/postman/launching-postman/postman-account.md:133]: Seeing this thread here now, I think contacting Support will do. I check on Zendesk for requests of users blocked by the ReCaptcha on that request reset page but it doesn't seem like we have seen any for a while.

[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/launching-postman/postman-account.md:133]: great thanks @cyrielle-hada

**Comment by SueSmith** [comment]
@claireriley am thinking we're probably good to go re the captcha thing here if everything else looks ok?

**Comment by claireriley** [comment]
@SueSmith yeah I think what was added is good, I'm outdated on this one 😆

**Pull request merged**